packages/@glimmer/babel-preset/index.js

```diff
@@ -75,9 +75,19 @@ module.exports = function (api, options) {
         { legacy: true },
       ],
 
-      __loadPlugins
-        ? require('@babel/plugin-proposal-class-properties')
-        : require.resolve('@babel/plugin-proposal-class-properties'),
+      [
+        __loadPlugins
+          ? require('@babel/plugin-proposal-class-properties')
+          : require.resolve('@babel/plugin-proposal-class-properties'),
+        { loose: true },
+      ],
+
+      [
+        __loadPlugins
+          ? require('@babel/plugin-proposal-private-methods')
+          : require.resolve('@babel/plugin-proposal-private-methods'),
+        { loose: true },
```

Comment:
Same as above

Reply:
This is going to emit `WeakSet` in strict mode. Are we ok with that?

Reply:
Not really, I don't think we can use `WeakSet` (IE11 doesn't support it and we don't directly polyfill the global).

Reply:
Ok well then this is going to have to default to `loose: true` because these plugins need to [share the options](https://babeljs.io/docs/en/babel-plugin-proposal-private-methods#loose).

Reply:
We can still default it to `false` and then, if you need to support IE11, allow users to set it to `true`. I'm, very hesitant to say the default is something that _will_ cause breaking failures when we eventually are no longer polyfilling (we have actually had failures caused by this both in Ember and elsewhere).

Reply:
I'm not really sure how to move forward here.